geode-core/src/main/java/org/apache/geode/management/internal/MBeanJMXAdapter.java

```diff
@@ -71,14 +69,17 @@
 
   private DistributedMember distMember;
 
-  private Logger logger = LogService.getLogger();
+  private Logger logger;
```

Comment:
usually a logger is static

Reply:
I spent quite a bit of time understanding why the logger should be static, and it appears to be a combination of convention and also protecting against potentially expensive instantiations if there are many instances of the object which references the logger.

https://logging.apache.org/log4j/2.x/manual/usage.html#StaticVsNonStatic

The downside of using a static logger is of course that you cannot inject the logger via DI.  Without DI, we can still test using facades, adapters, and other such tools but its not as straight forward as simple DI.

I went ahead and made it static and inlined the assignment, then added a package-protected wrapper method for the logger that I verify in my test.